jt-mosaic/src/main/java/it/geosolutions/jaiext/mosaic/MosaicOpImage.java

```diff
@@ -1260,7 +1260,7 @@ private void byteLoop(RasterBeanAccessor[] srcBeans, RasterAccessor dst) {
 
                         // If no alpha channel or Roi is present, the weight
                         // is set to 1 or 0 if the pixel has or not a No Data value
-                        if (hasNoData[s]) {
```

Comment:
Looks like "hasNoData" is not used any longer, should be removed

Reply:
Thanks for the hint.
I have removed that field, accordingly